src/berry_mill/kiwrap.py

```diff
@@ -85,6 +86,20 @@ def add_repo(self, reponame: str, repodata: Dict[str, str]) -> KiwiParent:
 
         return self
 
+    def _verify_gpg_key(self, key_path):
+        """
+        Virify wether the downloaded file is a GPG key
+        """
+        try:
+            result = subprocess.run(['gpg', '--import', key_path], capture_output=True, text=True)
+            if result.returncode == 0:
+                return True
+            else:
+                return False
```

Comment:
🤣  Because `return bool(subprocess.run(['gpg'.......]))` is too mainstream?

Reply:
done